internal/store/store.go

```diff
@@ -579,13 +609,16 @@ func (bs *BlockStore) Close() error {
 //---------------------------------- KEY ENCODING -----------------------------------------
 
 // key prefixes
+// NB: Before modifying these, cross-check them with those in
+// internal/state/store.go
```

Comment:
they should probably all be in one package, but this is a good note.

Reply:
Again, I apologise to @thanethomson for not having realized the collision in prefix that I introduced here

Reply:
Hey I didn't know about it either. It was a team effort to track down the source of the issue :slightly_smiling_face:

Reply:
I've added a TODO here to move these prefixes to a common package.